opm/io/eclipse/EclFile.hpp

```diff
@@ -38,12 +38,12 @@ class EclFile
     explicit EclFile(const std::string& filename, bool preload = false);
     bool formattedInput() const { return formatted; }
 
-    void loadData();                            // load all data
-    void loadData(const std::string& arrName);         // load all arrays with array name equal to arrName
-    void loadData(int arrIndex);                // load data based on array indices in vector arrIndex
-    void loadData(const std::vector<int>& arrIndex);   // load data based on array indices in vector arrIndex
+    void loadData() const;                                  // load all data
```

Comment:
this makes little sense. this is a loader, why do you want to make them const? seems just confusing and misleading.